SalesforceMobileSDK-iOS.podspec

```diff
@@ -74,7 +74,8 @@ Pod::Spec.new do |s|
       sdkcore.dependency 'SalesforceMobileSDK-iOS/SalesforceOAuth'
       sdkcore.dependency 'OpenSSL-Universal', '1.0.1.k'
       sdkcore.dependency 'SQLCipher', '~> 3.1'
```

Comment:
Should the ```SQLCipher``` dependency in the line above remain, or should it go, now that we have the ```SQLCipher/fts``` dependency?